src/stylesheets/components/_accordions.scss

```diff
@@ -36,7 +35,6 @@ $accordion-border: 3px solid $color-gray-lightest;
   button {
     background-color: $color-gray-lightest;
     background-image: url('../img/minus.png');
-    background-image: url('../img/minus.svg');
     background-position: right 3rem center;
```

Comment:
Omg I must have stared at these for like 2 minutes trying to figure out why every image was duplicated. Must have just had a moment of code blindness where the two lines looked exactly the same. Totally did not notice the file extension was different. Will revert now.

However, it does cause a linter warning, because it's a duplicate property. I guess there's no way around that...